src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -590,13 +590,25 @@ default int levelMinValue() {
         return 1;
     }
 
+    @ConfigItem(
+        keyName = "levelIntervalOverride",
+        name = "Interval Override Level",
+        description = "The minimum level after which all level ups send a notification.<br/>" +
```

Comment:
```suggestion
        description = "All level ups starting from this override level send a notification, disregarding any previously configured notify interval.<br/>" +
```

`after which` may imply an exclusive lower bound (while code does inclusive)

Reply:
```suggestion
        description = "All level ups starting from this override level send a notification, disregarding the configured Notify Interval.<br/>" +
```

Quick reading the previous suggestion without thinking about it explicitly mentioning the `notify interval` could make it sound like it ignores all config settings. (i.e. exactly what I thought on first test)